spec/models/post_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  context 'Post Model' do
+    before(:example) do
+      @user = User.create(name: 'Zeeshan', posts_counter: 0)
+    end
+
+    subject { @user.posts.new(title: 'Post', text: 'Lorem Ipsem', likes_counter: 0, comments_counter: 0) }
+    before { subject.save }
```

Comment:
- [ ] Kindly add a unit test for the `recent_comments` method you have in your Post model because it is a requirement to add unit tests for all custom methods in your models.